docroot/themes/contrib/civictheme/civictheme_library/components/00-base/layout/layout.scss

```diff
@@ -7,6 +7,10 @@
 
   &__sidebar {
     @include ct-print-hide();
+
+    @include ct-breakpoint-upto('m') {
+      margin-bottom: ct-spacing(2);
```

Comment:
@govindmaloo 
Layout component cannot have any vertical spacing added.

Please use other means to add the spacing within children components.